veggieseasons/lib/data/model.dart

```diff
@@ -0,0 +1,71 @@
+// Copyright 2018 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:scoped_model/scoped_model.dart';
+import 'package:veggieseasons/data/veggie.dart';
+import 'package:veggieseasons/data/local_veggie_provider.dart';
+
+class AppState extends Model {
```

Comment:
You seem to be using ScopedModel yet never actually changing the state. I suppose that's just because you haven't implemented that functionality yet. Otherwise, you might be better off just using `InheritedWidget`.

Reply:
Updates are coming. I have more of a complete app in https://github.com/redbrogdon/veggieseasons. I'm chunking it up while landing here to keep the PRs manageable.